js/handlers/buttonView.js

```diff
@@ -126,11 +121,11 @@ define([
 
             var measurements = this.$el.onscreen();
 
-            //this is to fix ios7 iphone4 miscalculation
+            // this is to fix ios7 iphone4 miscalculation
```

Comment:
considering the comment, is this code still required?

Reply:
@moloko, @oliverfoster was just looking at this earlier. would be handy to be able to configure the threshold at which the button is made visible

Reply:
cool, that's slightly beyond the scope of this one. i'll just change the comment for now and add a ticket https://github.com/adaptlearning/adapt_framework/issues/2256